jupyter-book/trajectories/pseudotemporal.ipynb

```diff
@@ -15,53 +15,95 @@
    ]
```

Comment:
<p>"Louvain <strong>[CITE]"</strong></p><p>Louvain is dead. Long live Leiden. (Please replace with Leiden).</p><p> </p><p>"partitions the graph into Louvain clusters"</p><p>It actually looks for Leiden first and if it's not present it attempts to use Louvain clusters. Please change to Leiden here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/multimodal-single-cell-best-practices/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Yes, I wrote Louvain because that's what is specified in the paper. I updated it.</p>